src/System.Globalization/tests/Common/UnicodeData.cslib

```diff
@@ -0,0 +1,19 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Why is all of this in Common?  Is it used by more than the System.Globalization tests?